tests/test_data_utils.py

```diff
@@ -0,0 +1,169 @@
+import pandas as pd
+import pytest
+
+from vivarium_census_prl_synth_pop.constants import metadata
+from vivarium_census_prl_synth_pop.results_processing.formatter import (
+    combine_joint_filers,
+    flatten_data,
+    format_1040_dataset,
+)
+
+
+@pytest.fixture(scope="module")
+def dummy_1040():
+    return pd.DataFrame(
```

Comment:
consider adding docstrings to these dummy datasets since they're kinda complicated and explain why you did what you did (i.e. what edge cases you're trying to test against)